account_asset_analytic/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Account asset analytic
+======================
+
+This module allows you to define an analytic account per asset and creates
+expenses moves with that analytic account.
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/87/8.0
+
+For further information, please visit:
```

Comment:
Remove this (and update your local template to https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)